src/task.rs

```diff
@@ -1003,13 +1012,17 @@ impl<'a> Runner<'a> {
         } else {
             0
         };
-        let target = if task.targets.len() == 0 {
+        let mut target = if task.targets.len() == 0 {
             "".to_string()
         } else if let Some(interpolate) = &job.interpolate {
             replace_interpolate(&task.targets[target_index], interpolate)
         } else {
             task.targets[target_index].clone()
         };
+        if target.len() != 0 {
+            target = target;
+        }
```

Comment:
What is this doing?

Reply:
it was a refactor thing, let me remove it.